frontend/.env

```diff
@@ -5,4 +5,4 @@
 REACT_APP_REST_API_BASE_URL=http://localhost:5000/rest-api
 
 #If set to true, then REACT_APP_REST_API_BASE_URL doesn't matter
-REACT_APP_MOCK_WITH_MSW=false
\ No newline at end of file
+REACT_APP_MOCK_WITH_MSW=true
```

Comment:
No i to tez chyba nie powinnismy commitowac? Bo teraz defauloto dziala z MSW. Tak samo dla swoich potrzeb mozesz z robic .env.local i nie commitowac.

Reply:
https://create-react-app.dev/docs/adding-custom-environment-variables/

Reply:
ok :) Przez przypadek! Racja, Pytanie jak wyżej!